### DISCUSSION TITLE
I am stuck

### DISCUSSION THREAD

**Discussion by TuanMV1** [codemagic-ci-cd/codemagic-docs#1483] [Q&A] [answered]
I am trying start build flutter application but status is queued even though there is only one process

![image](https://user-images.githubusercontent.com/124946938/225245912-c12a15ef-9610-4c93-bdae-8d2eba050ed4.png)

**Comment by himesh-cm** [comment, marked as answer]
Hi @TuanMV1, Please check the announcement [here](https://github.com/codemagic-ci-cd/codemagic-docs/discussions/1482)
